.projectile

```diff
@@ -0,0 +1 @@
+-/vendor
```

Comment:
I would defer to you as far as whether you think it's a good idea or not--I haven't really thought through the implications for other emacs users. I'm sure there is some way to configure projectile exclude paths in one's emacs configuration which would addres my problem more generally.